web/modules/custom/joinup_search/config/install/facets.facet_source.search_api_views__search__page_1.yml

```diff
@@ -0,0 +1,8 @@
+uuid: null
+langcode: en
+status: true
+dependencies: {  }
+id: search_api_views__search__page_1
+name: 'search_api_views:search:page_1'
+filter_key: null
+url_processor: query_string
```

Comment:
Why is this added? If this is necessary, can we give it a proper machine name? `page_1` is overly generic, it's not clear what this is used for.

On a minor remark, it is not common to include the `uuid` key and give it the value `null`. This is usually omitted. I don't think it is harmful though.